scripts/development/m3_stack/README.md

```diff
@@ -9,19 +9,42 @@ This docker-compose file will setup the following environment:
 
 ## Usage
 
-Use the `start_m3.sh` and `stop_m3.sh` scripts
+Use the `start_m3.sh` and `stop_m3.sh` scripts.
 
 ## Grafana
 
 Use Grafana by navigating to `http://localhost:3000` and using `admin` for both the username and password. The M3DB dashboard should already be populated and working.
 
+## Jaeger
+
+To start Jaeger, you need to set the environment variable `START_JAEGER` to `true` when you run `start_m3.sh`.
+
+```
+START_JAEGER=true ./start_m3.sh
```

Comment:
nit: should this be `USE_JAEGER` instead? cause we do have to stop it in `stop_m3.sh`